scrypto/src/resource/proof.rs

```diff
@@ -71,20 +46,82 @@ impl From<NonFungibleAddress> for ProofValidationMode {
     }
 }
 
-impl Clone for Proof {
-    native_methods! {
-        NativeMethod::Proof => {
-            fn clone(&self) -> Self {
-                ProofMethod::Clone,
-                ProofCloneInvocation {
-                    receiver: self.0
-                }
-            }
-        }
+pub trait SysProof {
+    fn sys_clone<Y, E: Debug + TypeId<ScryptoCustomTypeId> + Decode<ScryptoCustomTypeId>>(
```

Comment:
Looks like all these traits are intentional after the refactoring. Will skip for now and resort to a discussion.